apps/formbricks-com/app/docs/contributing/gitpod/page.mdx

```diff
@@ -1,15 +1,97 @@
+- Gitpod is preparing your workspace with all the necessary dependencies and configurations. You will see this page while Gitpod sets up your development environment.
+
+**Gitpod running the Workspace**
+<Image src={GitpodRunning} alt="Gitpod Running Workspace Page" quality="100" className="rounded-lg max-w-full sm:max-w-3xl" />
+- Once the workspace is fully prepared, voila, it enters the running state. You can start working on your project in this environment.
+
+### Ports and Services
+
+Here are the ports and corresponding URLs for the services within your Gitpod environment:
+
+- **Port 3000**:
+   - **Service**: Demo App
+   - **Description**: This port hosts the demo application of your project. You can access and interact with your application's demo by navigating to this port.
+
+- **Port 3001**:
+   - **Service**: Formbricks website
+   - **Description**: This port hosts the [Formbricks](https://formbricks.com) website, which contains documents, pricing, blogs, best practices, and concierge service.
+
+- **Port 3002**:
+   - **Service**: Formbricks In-product Survey Demo App
+   - **Description**: This app helps you test your in-app surveys. You can create and test user actions, create and update user attributes, etc.
+
+- **Port 5432**:
+   - **Service**: PostgreSQL Database Server
+   - **Description**: The PostgreSQL DB is hosted on this port.
+
+- **Port 1025**:
+   - **Service**: SMPT server
+   - **Description**: SMTP Server for sending and receiving email messages. This server is responsible for handling email communication.
+
+- **Port 8025**:
+   - **Service**: Mailhog
+
+### Accessing port URLs
+  1. **Direct URL Composition**:
+   - You can access the dedicated port URL by pre-pending the port number to the workspace URL.
+   - For example, if you want to access port 3000, you can use the URL format: `3000-yourworkspace.ws-eu45.gitpod.io`.
+
+  2. **Using `gp url` Command**:
+   - Gitpod provides a convenient command, `gp url`, to quickly retrieve the URL for a specific port.
+   - Simply use the command followed by the desired port number. For example, to get the URL for port 3000, run: `gp url 3000`.
+
+  3. **Listing All Open Port URLs**:
+   - If you prefer to see a list of all open port URLs at once, you can use the `gp ports list` command.
+   - Running this command will display a list of ports along with their corresponding URLs.
+
+  4. **Viewing All Ports in Panel**:
+   - Gitpod also offers a user-friendly 'Ports' tab in the Gitpod panel.
+   - Click on the 'Ports' tab to view a list of all open ports and their respective URLs.
+
+  <Image src={GitpodPorts} alt="Gitpod Ports tab" quality="100" className="rounded-lg max-w-full sm:max-w-3xl" />
+
+  These URLs and port numbers represent various services and endpoints within your Gitpod environment. You can access and interact with these services by the Port URL for the respective service.
```

Comment:
We end most of our guides with "Still can’t figure it out?: [Join our Discord!](https://formbricks.com/discord)", so can you do that as well here please?

It comforts the user to reach out even if they follow the steps but don't reach the expected output.

Reply:
yup, that would be great, adding it